cyber/proto/dag_conf.proto

```diff
@@ -7,11 +7,17 @@ import "cyber/proto/component_conf.proto";
 message ComponentInfo {
   optional string class_name = 1;
   optional ComponentConfig config = 2;
+  optional uint64 adc_index = 3;
+  optional string episode_name = 4;
+
```

Comment:
No additional line here. Try `./scripts/clang-format.sh` to do auto-format and learn from it :)